Library/Homebrew/cmd/bottle.rb

```diff
@@ -300,69 +301,78 @@ def bottle_formula(f)
     puts "./#{filename}"
     puts output
 
-    if ARGV.include? "--rb"
-      File.open("#{filename.prefix}.bottle.rb", "w") do |file|
-        file.write("\# #{f.full_name}\n")
-        file.write(output)
+    if ARGV.include? "--json"
+      json = {
+        f.full_name => {
+          "formula" => {
+            "pkg_version" => f.pkg_version.to_s,
+            "path" => f.path.to_s.gsub("#{HOMEBREW_REPOSITORY}/", ""),
+          },
+          "bottle" => {
+            "disabled?" => f.bottle_disabled?,
+            "disable_reason" => f.bottle_disable_reason,
```

Comment:
👍. Need to do more testing here around edge-cases like that to figure out what I want to do.